src/encord_active/lib/embeddings/cnn.py

```diff
@@ -198,7 +198,7 @@ def generate_cnn_classification_embeddings(iterator: Iterator) -> List[LabelEmbe
             and collection["frame"] == iterator.frame
         ]
 
-        if not len(image_collections):
+        if not len(matching_image_collections):
```

Comment:
Could even just do `if not matching_image_collections` right?

Reply:
True 👍